text-zoom/android/src/main/java/com/capacitorjs/plugins/textzoom/TextZoomPlugin.java

```diff
@@ -0,0 +1,59 @@
+package com.capacitorjs.plugins.textzoom;
+
+import android.app.Activity;
+import android.os.Handler;
+import android.os.Looper;
+import android.webkit.WebView;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.NativePlugin;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+
+@NativePlugin(name = "TextZoom")
+public class TextZoomPlugin extends Plugin {
+    private TextZoom tz;
```

Comment:
I like these short names in the plugins but it doesn't seem consistent to have `tz` and `webView` and `activity` etc. I'm thinking it would be cleaner to use full words. I think in a couple we used "implementation" so far but `textZoom` is fine I think?

Reply:
Yeah I mostly used implementation because it was easier and a good way to show the plugin author intent. `tz` was laziness, good call